chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/impl/ProductDAOImpl.java

```diff
@@ -223,6 +234,15 @@ private void update(ProductEntity entity) {
 		
 	}
 	
+	private List<ProductEntity> getAllEntitiesIncludingDeleted() {
+		List<ProductEntity> result = entityManager.createQuery( "from ProductEntity pe "
+				+ "LEFT OUTER JOIN FETCH pe.developer "
+				+ "LEFT OUTER JOIN FETCH pe.productCertificationStatuses ", 

```

Comment:
Similar to Developer; not sure why joining in productCertificationStatuses object